src/item.cpp

```diff
@@ -1508,6 +1508,14 @@ void item::basic_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         info.push_back( iteminfo( "BASE", _( "Category: " ),
                                   "<header>" + get_category().name() + "</header>" ) );
     }
+    if( !type->weapon_category.empty() && parts->test( iteminfo_parts::WEAPON_CATEGORY ) ) {
+        const std::string weapon_categories = enumerate_as_string( type->weapon_category.begin(),
+        type->weapon_category.end(), [&]( const weapon_category_id & elem ) {
+            return elem->name().translated();
+        }, enumeration_conjunction::none );
+        info.push_back( iteminfo( "BASE", _( "Weapon Categories: " ),
```

Comment:
I think singular would be better here for consistency, UI uses singular for nearby `Category` and `Material` entries (even when there are multiple materials)
```suggestion
        info.push_back( iteminfo( "BASE", _( "Weapon Category: " ),
```